bindings/python/limbo/__init__.py

```diff
@@ -14,16 +14,16 @@
 )
 
 __all__ = [
-    "__version__",
-    "Connection",
-    "Cursor",
-    "InterfaceError",
-    "DatabaseError",
-    "DataError",
-    "OperationalError",
-    "IntegrityError",
-    "InternalError",
-    "ProgrammingError",
-    "NotSupportedError",
-    "connect",
+    '__version__',
+    'Connection',
+    'Cursor',
+    'InterfaceError',
+    'DatabaseError',
+    'DataError',
+    'OperationalError',
+    'IntegrityError',
+    'InternalError',
+    'ProgrammingError',
+    'NotSupportedError',
+    'connect',
```

Comment:
I would like to argue against use of single quotes. In Python it doesn't matter of course, but Black formatter [prefers double quotes](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings). Also in Rust we use double quotes. These are why I would prefer double quotes in this project.